packages/components/src/components/DistributionsChart/index.tsx

```diff
@@ -71,7 +81,7 @@ const InnerDistributionsChart: FC<{
       context.clearRect(0, 0, width, height);
 
       const getColor = (i: number) =>
-        isMulti ? d3.schemeCategory10[i] : "#5ba3cf";
+        isMulti ? d3.schemeCategory10[i] : "#6cabd2";
```

Comment:
This was true even before this PR, but with this lighter color it's more noticeable: function chart color has a different hue and saturation from dist chart color.

I think it'd be good if they were more consistent:
<img width="721" alt="Captura de pantalla 2023-06-15 a la(s) 15 29 33" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/bae7b33a-2329-4c66-88e6-4776fd2b56c7">